macos-limits.md

```diff
@@ -134,22 +134,23 @@ Unfortunately many features in macOS are outright unsupported with AMD and many
 
 GPU support becomes much more complicated due to the near-infinite amount of GPUs on the market, but the general breakdown is as follows:
 
-* AMD's GCN based GPUs are supported in the latest versions of macOS
+* AMD's GCN based GPUs and later are supported in the latest versions of macOS
   * AMD APUs are not supported however
   * AMD's [Lexa based cores](https://www.techpowerup.com/gpu-specs/amd-lexa.g806) from the Polaris series are also not supported
   * Special note for MSI Navi users: [Installer not working with 5700XT #901](https://github.com/acidanthera/bugtracker/issues/901)
-    * This issue is no longer present in macOS 11 (Big Sur).
-* Nvidia's GPU support is complicated:
-  * [Maxwell(9XX)](https://en.wikipedia.org/wiki/GeForce_900_series) and [Pascal(10XX)](https://en.wikipedia.org/wiki/GeForce_10_series) GPUs are limited to macOS 10.13: High Sierra
-  * [Nvidia's Turing(20XX,](https://en.wikipedia.org/wiki/GeForce_20_series)[16XX)](https://en.wikipedia.org/wiki/GeForce_16_series) GPUs are **not supported in any version of macOS**
-  * [Nvidia's Ampere(30XX)](https://en.wikipedia.org/wiki/GeForce_30_series) GPUs are **not supported in any version of macOS**
-  * [Nvidia's Kepler(6XX,](https://en.wikipedia.org/wiki/GeForce_600_series)[7XX)](https://en.wikipedia.org/wiki/GeForce_700_series) GPUs are supported up to macOS 11: Big Sur
+    * This issue is no longer present in macOS 11 (Big Sur) and later.
+* NVIDIA's GPU support is complicated:
+  * [Maxwell (900 series)](https://en.wikipedia.org/wiki/GeForce_900_series) and [Pascal (10 series)](https://en.wikipedia.org/wiki/GeForce_10_series) GPUs are limited to macOS 10.13: High Sierra
+  * [Turing (20 series,](https://en.wikipedia.org/wiki/GeForce_20_series)[16 series)](https://en.wikipedia.org/wiki/GeForce_16_series) GPUs are **not supported in any version of macOS**
```

Comment:
Maybe do `2000 series` instead. `20 series` sounds a little weird.

Reply:
The latest 3 gens are actually called the 10, 20, and 30 series. [Wikipedia](https://en.wikipedia.org/wiki/GeForce_30_series), [NVIDIA driver page](https://www.nvidia.com/download/index.aspx)

Reply:
Oh, fun
This is fine then